pkg/internal/cluster/logs/logs.go

```diff
@@ -114,24 +116,18 @@ func Collect(logger log.Logger, nodes []nodes.Node, dir string) error {
 
 // dumpDir dumps the dir nodeDir on the node to the dir hostDir on the host
 func dumpDir(logger log.Logger, node nodes.Node, nodeDir, hostDir string) (err error) {
-	// make tempdir to rsync nodeDir into (rsync handles taking a snapshot better)
-	tmp, err := mktemp(node)
-	if err != nil {
-		return err
-	}
-	defer func() {
-		if rerr := node.Command("rm", "-rf", tmp).Run(); rerr != nil && err == nil {
-			err = rerr
-		}
-	}()
-
-	// rsync into the temp dir
-	if err := node.Command("rsync", "--archive", path.Clean(nodeDir)+"/", tmp).Run(); err != nil {
-		return err
-	}
+	cmd := node.Command(
+		"sh", "-c",
+		// Tar will exit 1 if a file changed during the archival.
+		// We don't care about this, so we're invoking it in a shell
+		// And masking out 1 as a return value.
+		// Fatal errors will return exit code 2.
+		// http://man7.org/linux/man-pages/man1/tar.1.html#RETURN_VALUE
+		fmt.Sprintf(
+			`tar --hard-dereference -C %s -chf - . || (r=$?; [ $r -eq 1 ] || exit $r)`,
+			shellescape.Quote(path.Clean(nodeDir)+"/")),
```

Comment:
nit city: normally would balance the closing paren to it's own line, but gofmt seems ok with this

Reply:
done